_data/people.yml

```diff
@@ -42,14 +42,17 @@
     url: "lg"
     role: "PhD Candidate, US NRC Fellow"
     image: lewis.png
-  
 
 - group: 'Undergraduate Students'
   people:
   - name: "Bowen Quan"
     url: "bq"
     role: "Undergraduate Researcher"
     image: Bowen_Quan.jpg
+  - name: "Syn Hubbard"
+    url: "synh"
+    role: "Undergraduate Researcher"
+    image: synH.png
```

Comment:
We generally add people chronologically, so best to move this to the end of the undergrad section

Reply:
Changed!